src/06-products/products.apib

```diff
@@ -24,3 +24,22 @@ Get a list of products.
                         + name: `cookies` (string)
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
+
+### products.info [GET /products.info]
+
+Get details for a single product.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string)
```

Comment:
Q: since `name`, `code` and `description` are optional should they be nullable here as well?

Reply:
Fair enough I will change them.